src/services/namingService.ts

```diff
@@ -26,6 +26,10 @@ export class AzureNamingService {
       this.createShortStageName(stage),
     ].join("-");
 
+    if(includeHash && config.provider.resourceGroup) {
```

Comment:
I think if `config.provider.resourceGroup` is not defined, there are bigger problems here. It would probably be good to throw an error as opposed to just skipping the addition of the hash